tests/v2/Controller/ProController/TogglePermission.php

```diff
@@ -79,4 +78,54 @@ public function provideTestToggleBeneficiaryManagement(): ?\Generator
         yield 'Should toggle beneficiary management permission' => [MembreCentre::MANAGE_BENEFICIARIES_PERMISSION];
         yield 'Should toggle pro management permission' => [MembreCentre::MANAGE_PROS_PERMISSION];
     }
+
+    public function testCanNotSeeNotOwnedPermission(): void
+    {
+        // Test only for beneficiary management as pro list is unreachable without pro management permission
+        $testedPermission = MembreCentre::MANAGE_BENEFICIARIES_PERMISSION;
+        $ownedPermission = MembreCentre::MANAGE_PROS_PERMISSION;
+
+        self::ensureKernelShutdown();
+        $client = self::createClient();
+        $user = UserFactory::findByEmail(MemberFixture::MEMBER_MAIL_WITH_RELAYS_SHARED_WITH_MEMBER)->object();
+        $client->loginUser($user);
+
+        $relay = $user->getSubjectMembre()->getCentres()->first();
+        $userRelay = $user->getUserRelay($relay);
+
+        $userRelay->removePermission($testedPermission);
+        $this->getEntityManager()->flush();
+
```

Comment:
It looks like this could be extracted to be reused in `testCanNotGiveNotOwnedPermission()`